distro/adaptation/ubuntu-22.04

```diff
@@ -3,3 +3,4 @@ libunwind8: libunwind-14
 libunwind-dev: libunwind-14-dev
 libperl5: libperl5.34
 perl-modules-5: perl-modules-5.34
+python-setuptools: python3-setuptools
```

Comment:
thanks, in change log, can you provide the initial errors, and the steps that trigger the error, so to know what compiling issue is actually fixed

Reply:
i'm not able reproduce the error on my system anymore. Will close this pull request for now and raise a new PR if manage to reproduce on next system.